properties.schema

```diff
@@ -106,7 +105,7 @@
       "title": "Cycle through item pagination",
       "inputType": "Checkbox",
       "validators": [],
-      "help": "If enabled, the items in the open popup will be allowed to cycle through continuously"
+      "help": "If enabled, the popup navigation buttons will cycle continuously through the popup items (i.e. clicking next on the last item will take them to the first one)."
```

Comment:
Can we replace 'them' with 'the user' here?

Reply:
yep my bad I was actually trying to write this in such a way that it didn't reference the learner/user but forgot to adjust this bit. How about 'clicking next on the last item will cause the first item to be shown'?

Reply:
Perfect.

Reply:
Does this need doing still?

Reply:
it does, good spot